src/item.cpp

```diff
@@ -1897,18 +1897,30 @@ std::string item::tname( unsigned int quantity, bool with_prefix ) const
             ret << _("Bug");
         }
     }
+    // Wrap all armor mods together
+    const bool has_armor_mod = item_tags.count("wooled") + item_tags.count("furred") +
+        item_tags.count("leather_padded") + item_tags.count("kevlar_padded") > 0;
+    if( has_armor_mod ) {
+        ret << " (";
+    }
+    // It's probably hard to translate a single letter
+    // Any good way of doing _("W")[0] that aren't as hacky as _("W")[0]?
```

Comment:
How about just item_name + like with weapon mods, and only display the details when examining the item?